modules/default/calendar/calendar.js

```diff
@@ -744,6 +765,16 @@ Module.register("calendar", {
 		return this.getCalendarProperty(url, "color", "#fff");
 	},
 
+	/**
+	 * Retrieves the bgColor for a specific calendar url.
+	 *
+	 * @param {string} url The calendar url
+	 * @returns {string} The color
+	 */
+	bgColorForUrl: function (url) {
```

Comment:
this is only used in one place, why create a method for this then?

Reply:
I'll make adjustments and respectively update my code